### ISSUE TITLE
Bash completion does't work

### ISSUE THREAD

**Issue by realnc** [user-none/KDocker#31]
(KDocker 5.1)

The bash completion file does not seem to work. For example, typing:

```
kdocker -
```

And hitting TAB does not do anything.

In case it matters, this warning is shown (on Gentoo) with bash-completion 2.7:

```
Problems with installed bash completions were found:
kdocker: 'have' command is deprecated and must not be used.
```

It appears `have` has been deprecated back in 2.1 or so.

**Comment by Daxx** [comment]
As far as I can discover, the solution is simply to remove 'have'.
Options after `kdocker -[TAB]` are not listed but never were (?) because they're all single character.
Pasting this into a terminal allows command completion for, say, `kdocker fir[TAB]` (-› firefox) which might have been all that was intended.

```bash
_kdocker()
{
        local cur

        COMPREPLY=()
        cur=${COMP_WORDS[COMP_CWORD]}
        COMPREPLY=($( compgen -A command $cur ) )
}

complete -F _kdocker kdocker

```
This seems a better solution because it's wrong, currently, as you point out.
Comments welcome.

[https://wiki.gentoo.org/wiki/Bash/Installing_completion_files](url)
[https://www.gnu.org/software/bash/manual/html_node/Programmable-Completion-Builtins.html](url)
(Some may need to uncomment a section in /etc/bash.bashrc), following:
`# enable bash completion in interactive shells
`

**Comment by user-none** [comment]
Fixed in 4570cea.
